client/src/planwise/client/projects2/components/settings.cljs

```diff
@@ -54,6 +54,17 @@
        "number" [common2/numeric-field (assoc props :on-change change-fn)]
        [common2/text-field props]))))
 
+(defn- current-project-checkbox
+  [label path checked-value unchecked-value other-props]
+  (let [current-project (rf/subscribe [:projects2/current-project])
+        checked         (not= (get-in @current-project path) unchecked-value)
```

Comment:
Why not `(= (get-in @current-project path) checked-value)` ?

Reply:
I will update!